src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -1607,7 +1607,9 @@ private sealed class NoOpScaleMonitor : IScaleMonitor
             /// <param name="name">A descriptive name.</param>
             public NoOpScaleMonitor(string name)
             {
+#pragma warning disable CS0618 // Type or member is obsolete
                 this.Descriptor = new ScaleMonitorDescriptor(name);
+#pragma warning restore CS0618 // Type or member is obsolete
             }
 
             /// <summary>

```

Comment:
this is the other pragma I added. All other diffs are just from pulling from `dev`